app/controllers/setting_controller.rb

```diff
@@ -70,35 +72,36 @@ def create
             next
           end
           logic(available_items)
-          next_customer_time += exponential(60 / $consumer_per_minute, random_generator)
```

Comment:
So Ruby converted $consumerperminute implicitly to int before dividing?

Reply:
The request from the management UI sets $consumer_per_minute to int 100.
That's why both sides of the division were ints.